src/requestTypeIdentifier.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+/**
+* @file
+* requestType indentifier
+*
+* This file contains functions designed to find out which endpoints
+* needs to be called, based on the provided filter fields.
+*
+* Example of usage:
+*
+*   let typeID = to.makeTypeID('doc/work-context.jsonld');
+*
+*   let retval = typeID.getType('isAnalysisOf');
+*   let isval = typeID.isType('isAnalysisOf', 'relations');
+*
+*/
+import fs from 'fs';
+import {die} from './utils.js';
+
+
+/**
+* requestType enum. Only these request types are supported.
+*/
+export let requestType = {
+  RELATIONS: 'relations',
+  MOREINFO: 'moreinfo',
+  BRIEFDISPLAY: 'briefdisplay',
+  COLLECTION: 'collection',
+  DKABM: 'dkabm'
+};
+
+/**
+* private function. validates type
+*/
+function isRequestType(type) {
+  for (var key in requestType) {
+    if (type === requestType[key]) {
+      return true;
+    }
+    return false;
+  }
```

Comment:
Could be written simpler, ie.

      if (type === requestType[key]) {
       return true;
     }
     return false;

is the same as

    return type === requestType[key];